### PR TITLE
Use isInteger()

### PR DESCRIPTION
**Pull request by yguedidi**
rector green

### CODE REVIEW

File: packages/BetterPhpDocParser/PhpDocManipulator/PhpDocTypeChanger.php [hunk 1]
```diff
@@ -44,6 +45,7 @@ final class PhpDocTypeChanger
         SpacingAwareArrayTypeNode::class,
         SpacingAwareCallableTypeNode::class,
         ArrayShapeNode::class,
+        ConstTypeNode::class,
```
Comment (yguedidi) [inline packages/BetterPhpDocParser/PhpDocManipulator/PhpDocTypeChanger.php]: this is the fix to the regressions, isn't it more a bug fix? 🙂

Comment (yguedidi) [reply packages/BetterPhpDocParser/PhpDocManipulator/PhpDocTypeChanger.php]: ~Found a better fix I think, back to draft~ pushed

**Comment by samsonasik** [comment]
Can you rebase?

**Comment by yguedidi** [comment]
Was on it, just did :)

**Comment by samsonasik** [comment]
let's give it a try, thank you.

**Pull request merged**